src/main/java/com/fasterxml/jackson/datatype/jsr310/ser/OffsetDateTimeSerializer.java

```diff
@@ -8,6 +9,7 @@
 
     protected OffsetDateTimeSerializer() {
         super(OffsetDateTime.class, dt -> dt.toInstant().toEpochMilli(),
-                OffsetDateTime::toEpochSecond, OffsetDateTime::getNano);
+                OffsetDateTime::toEpochSecond, OffsetDateTime::getNano,
+                dt -> DateTimeFormatter.ISO_OFFSET_DATE_TIME.format(dt));
```

Comment:
You don't need the lambda here. You can replace:

`dt -> DateTimeFormatter.ISO_OFFSET_DATE_TIME.format(dt)`

With:

`DateTimeFormatter.ISO_OFFSET_DATE_TIME::format`